package.json

```diff
@@ -62,20 +60,19 @@
         "cypress-axe": "^1.4.0",
         "file-loader": "^6.2.0",
         "filemanager-webpack-plugin": "^8.0.0",
-        "husky": "^6.0.0",
```

Comment:
We used to use this to run pre-commit hooks. This automated local linting and POT file updates. I feel like it could still be a helpful tool.